README.md

```diff
@@ -53,6 +54,10 @@ and also follows [Kaldi](http://kaldi-asr.org/) style data processing, feature e
 - Transformer based end-to-end ST (new!)
 - Transformer based end-to-end MT (new!)
 
+### VC: Voice conversion
+- End-to-end VC based on cascaded ASR+TTS (new!)
+- Baseline system for Voice Conversion Challenge 2020!
+
```

Comment:
Like the TTS function, you can add VC results (audio example link) and pre-trained models in this main README.md (advertise your work!).
Probably it would be better to have an additional cool audio example webpage, but you can do it later.